lib/src/base-styles/_nx-page.scss

```diff
@@ -23,25 +25,41 @@
     white-space: nowrap;
   }
 
-  .nx-btn-bar {
+  .nx-btn-bar, .nx-dropdown {
```

Comment:
I believe the correct way was to have the dropdown inside of a btn-bar

Reply:
> I believe the correct way was to have the dropdown inside of a btn-bar

I had it that way and then I second guessed myself. Reverted here: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/32969860b6a0d8b1e41d5cd37d0a8c7ab2af55d0